src/Screens/Explore/Reviews.tsx

```diff
@@ -277,12 +268,14 @@ const styles = StyleSheet.create({
   activityType: {
     color: '#FFF',
     textTransform: 'capitalize',
+    textAlign: 'center',
   },
   activityTypeContainer: {
     backgroundColor: '#0B94EF',
     borderRadius: 5,
-    paddingVertical: 3.5,
-    paddingHorizontal: 5.5,
+    paddingVertical: 2,
+    paddingHorizontal: 8,
+    alignSelf: 'flex-start',
```

Comment:
this was the issue with the scuba/snorkel tags. it was dependent on the width of the author's name

Reply:
Really? Interesting.

Reply:
yeah it's a flex box thing